application/deed/utils.py

```diff
@@ -0,0 +1,32 @@
+import os
+from jsonschema.validators import validator_for
+import json
+
+
+def validate_helper(json_to_validate):
+    error_message = ""
+    error_list = sorted(_title_validator.iter_errors(json_to_validate),
+                        key=str, reverse=True)
+
+    for count, error in enumerate(error_list, start=1):
+        error_message += "Problem %s:\n\n%s\n\n" % (count, str(error))
+
+    return len(error_list), error_message
+
+
+def _load_json_schema():
+    script_path = os.path.dirname(os.path.realpath(__file__))
```

Comment:
Discussion: I hate that we are finding paths relative to files that we are in. I think it was OK for ALPHA but for BETA I don't like it.

I think it would make it a hell of a lot more readable if you find the app root
```
app_dir = os.path.split(os.path.abspath(os.path.dirname(__file__)))[0]
```
and find things relative to that. Maybe its just me. What do you think?